<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_ui_message">
    <sys_ui_message action="INSERT_OR_UPDATE">
        <application/>
        <code/>
        <commented_message/>
        <key>Azure instance {0} has been successfully terminated with errors in the after deprovision workflow.</key>
        <language>uk</language>
        <message>Екземпляр Azure {0} успішно зупинено з помилками у процесі після видалення.</message>
        <sys_class_name>sys_ui_message</sys_class_name>
        <sys_created_by>bogdan.felinskyi</sys_created_by>
        <sys_created_on>2022-03-23 17:36:52</sys_created_on>
        <sys_id>aeae30271b96c910c069657bab4bcbfd</sys_id>
        <sys_mod_count>0</sys_mod_count>
        <sys_name>Azure instance {0} has been successfully terminated with errors in the after deprovision workflow.</sys_name>
        <sys_package display_value="I18N: Ukrainian Translations" source="176ebda287b38910f4b131570cbb3551">176ebda287b38910f4b131570cbb3551</sys_package>
        <sys_policy/>
        <sys_scope display_value="I18N: Ukrainian Translations">176ebda287b38910f4b131570cbb3551</sys_scope>
        <sys_update_name>sys_ui_message_aeae30271b96c910c069657bab4bcbfd</sys_update_name>
        <sys_updated_by>bogdan.felinskyi</sys_updated_by>
        <sys_updated_on>2022-03-23 17:36:52</sys_updated_on>
    </sys_ui_message>
</record_update>
